wallet/reference/non-evm-apis/starknet-snap-api.md

```diff
@@ -0,0 +1,830 @@
+---
+description: See the Starknet Snap API reference.
+sidebar_position: 1
+toc_max_heading_level: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Starknet Snap API
+
+With the [Starknet Snap](../../how-to/use-non-evm-networks/starknet/index.md) installed, dapps can
+use the Starknet Snap API to interact with users' Starknet accounts (for example, to send transactions).
+
+:::note
+You can also communicate with the Starknet network using the
```

Comment:
```suggestion
To read from the Starknet network (get transactions, get account information, ...) you can use the
```

Reply:
We could give some information regarding the chain id. 

The current starknet network has 2 public network. One testnet and the mainnet. The corresponding chain ids are : 

```javascript
SN_MAIN = "0x534e5f4d41494e"
SN_SEPOLIA = "0x534e5f5345504f4c4941"
```